base_comment_template/security/security.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model="ir.rule" id="rule_multicompany">
+        <field name="name">Base comment multi-company</field>
+        <field name="model_id" ref="model_base_comment_template" />
+        <field name="global" eval="True" />
+        <field name="domain_force">
+             ['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
```

Comment:
This rules is incorrect. See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0:
```suggestion
             ['|',('company_id','=',False),('company_id','in',company_ids)]</field>
```